Userland/Libraries/LibC/unistd.h

```diff
@@ -36,7 +36,7 @@ int get_process_name(char* buffer, int buffer_size);
 int set_process_name(char const* name, size_t name_length);
 void dump_backtrace(void);
 int fsync(int fd);
-int sysbeep(size_t tone);
+int sysbeep(size_t tone, size_t milliseconds_duration);
```

Comment:
Is this even a standardized function? I can't find anything online, so maybe we should just move it to `LibCore` completely.